contrib/functions/go/delete-annotations/README.md

```diff
@@ -0,0 +1,34 @@
+# delete-annotations
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Deletes the supplied annotation keys from the resource(s) in a package. It can delete multiple annotations provided as a comma separated string as part of the function config (usage instructions below).
+
+<!--mdtogo:Long-->
+
+## Usage
+
+The function will execute as follows:
+
+1. Searches for resources with valid metadata
+2. Deletes the annotation keys provided in KptFile or imperatively as config parameters
+
+To execute imperatively:
+```shell
+$ kpt fn eval -i gcr.io/kpt-fn-contrib/delete-annotations:unstable -- annotationKeys=annotation.to.delete,another.annotation.to.delete
+```
+
+To execute `delete-annotations` declaratively include the function in kpt package pipeline as follows:
+```yaml
+...
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn-contrib/delete-annotations:unstable
+      configMap:
+        annotationKeys: annotation.to.delete,another.annotation.to.delete
```

Comment:
Does k8s allow `.` in annotations?
It's more common to use dash `-`. If you agree, please update other places as well.

Reply:
k8s allows `.` in annotations. I've verified that.

Reply:
Got it.
I'd still suggest to make use `-` in between which is more common. Otherwise, when you mention it in the doc, it looks like a path.